src/dds/rsdds-device-factory.cpp

```diff
@@ -143,7 +143,7 @@ rsdds_device_factory::rsdds_device_factory( std::shared_ptr< context > const & c
                       .default_value( 5 );
             domain.query_devices_min
                 = dds_settings.nested( std::string( "query-devices-min", 17 ), &json::is_number_integer )
```

Comment:
General questions
Any idea what -min means?
minimum timeout?
What is 17?
What is 3-->0 used for?

Reply:
- min is the time the listener waits for new dds participant 
- 17 is just the number of characters in the "query-devices-min" string - only syntax needed for the nested method
- 3-->0 has been tried so that the listener will not wait 3 seconds for nothing
See ticket for reason why this PR has been changed to draft.